test/utils/update/beneficial_owners_managing_officers_data_fetch.spec.ts

```diff
@@ -34,6 +41,26 @@ describe("util beneficial owners managing officers data fetch", () => {
     expect(appData.update?.bo_mo_data_fetched).toBe(true);
   });
 
+  test("Should not set BO data in appData if no Company PSCs returned", async () => {
+    appData = { "transaction_id": "123", "overseas_entity_id": "456" };
+    mockGetCompanyPscService.mockReturnValue({ "items": [] });
+    await retrieveBeneficialOwners(req, appData);
+
+    expect(appData.beneficial_owners_individual?.[0].first_name).toEqual(undefined);
+    expect(appData.beneficial_owners_corporate?.[0].name).toEqual(undefined);
+    expect(appData.beneficial_owners_government_or_public_authority?.[0].name).toEqual(undefined);
```

Comment:
Should this be checking the update object?
```suggestion
    expect(appData.update?.review_beneficial_owners_individual?.[0].first_name).toEqual(undefined);
    expect(appData.update?.review_beneficial_owners_corporate?.[0].name).toEqual(undefined);
    expect(appData.update?.review_beneficial_owners_government_or_public_authority?.[0].name).toEqual(undefined);
```